src/Tribe/Views/V2/Widgets/Widget_List.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * List Widget
+ *
+ * @package Tribe\Events\Views\V2\Widgets
+ * @since   TBD
+ */
+
+namespace Tribe\Events\Views\V2\Widgets;
+
+use Tribe\Widget\Widget_Abstract;
+
+/**
+ * Class for List Widget.
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Views\V2\Widgets
+ */
+class Widget_List extends Widget_Abstract {
+
+	/**
+	 * {@inheritDoc}
+	 */
+	protected $slug = 'tribe_events_list_widget';
+
+	/**
+	 * {@inheritDoc}
+	 */
+	protected $view_slug = 'widget-list';
+
+	/**
+	 * {@inheritDoc}
+	 */
+	protected $default_arguments = [
+		// View options.
+		'view'                 => null,
+		'should_manage_url'    => false,
+
+		// Event widget options.
+		'id'                   => null,
+		'alias-slugs'          => null,
+		'title'                => '',
+		'limit'                => 5,
+		'no_upcoming_events'   => '',
+		'featured_events_only' => false,
+		'tribe_is_list_widget' => true,
+		'jsonld_enable'        => true,
+
+		// WP_Widget properties.
+		'id_base'              => 'tribe-events-list-widget',
+		'name'                 => null,
+		'widget_options'       => [
+			'classname'   => 'tribe-events-list-widget',
+			'description' => null,
+		],
+		'control_options'      => [
+			'id_base' => 'tribe-events-list-widget',
+		],
+	];
+
+	/**
+	 * {@inheritDoc}
+	 */
+	protected $validate_arguments_map = [
+		'should_manage_url'    => 'tribe_is_truthy',
+		'no_upcoming_events'   => 'tribe_is_truthy',
+		'featured_events_only' => 'tribe_is_truthy',
+		'jsonld_enable'        => 'tribe_is_truthy',
+	];
+
+	/**
+	 * @todo update in TEC-3612 & TEC-3613
+	 *
+	 * {@inheritDoc}
+	 */
+	public function setup() {
+
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function get_arguments() {
+		$arguments = $this->arguments;
+
+		$arguments['description']                   = esc_html__( 'A widget that displays upcoming events.', 'the-events-calendar' );
+		// todo update name once this widget is ready to replace the existing list widget.
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 19 spaces

<sub>hash: 1450c86c2f86026bb900c1cf74fbb86d</sub>